icu4c/source/common/dictbe.cpp

```diff
@@ -17,14 +17,20 @@
 #include "dictbe.h"
 #include "unicode/uniset.h"
 #include "unicode/chariter.h"
+#include "unicode/resbund.h"
 #include "unicode/ubrk.h"
+#include "ubrkimpl.h"
 #include "utracimp.h"
 #include "uvectr32.h"
 #include "uvector.h"
 #include "uassert.h"
 #include "unicode/normlzr.h"
 #include "cmemory.h"
 #include "dictionarydata.h"
+#include "utrie2.h"
+#include "uprops.h"
+#define INCLUDED_FROM_UCHAR_C
+#include "uchar_props_data.h"
```

Comment:
@markusicu  I am concern of this usage. Any opinion? should we just use UnicodeSet to replace such usage instead?